js/atree_constants.js

```diff
@@ -7001,6 +7001,15 @@ const atrees = {
                             ]
                         }
                     ]
+                },
+                {
+                    "type": "add_spell_prop",
+                    "base_spell": 1,
+                    "target_part": "Total Damage",
+                    "hits": {
+                        "Per Hit": 1
+                    },
+                    "display": "Total Damage"
```

Comment:
Fold this into `replace_spell` parts declaration -- no need to add it separately, also this will fk with lacerate